ShareClient/ShareClient.swift

```diff
@@ -69,18 +75,27 @@ private func dexcomPOST(_ url: URL, JSONData: [String: AnyObject]? = nil, callba
 public class ShareClient {
     public let username: String
     public let password: String
-
+    
+    private let shareServer:String?
```

Comment:
This should not be an optional. Then you can remove the forced unwrapping later (the exclamation points)